lua/starfall/libs_sv/constraint.lua

```diff
@@ -174,7 +193,20 @@ function constraint_library.ballsocket(e1, e2, bone1, bone2, v1, force_lim, torq
 	end
 end
 
---- Advanced Ballsocket two entities
+--- Ballsocket two entities together with more options
+-- @param e1 The first entity
+-- @param e2 The second entity
+-- @param bone1 Number bone of the first entity
+-- @param bone2 Number bone of the second entity
+-- @param v1 Position on the first entity, in its local space coordinates
+-- @param v2 Position on the second entity, in its local space coordinates
+-- @param force_lim Amount of force until it breaks, 0 = Unbreakable
+-- @param torque_lim Amount of torque until it breaks, 0 = Unbreakable
+-- @param minv Vector defining minimum rotation angle, local to the constraint
```

Comment:
I think this angle limits on world axes. Need to verify.

Reply:
World axis is correct